spectator-ext-jvm/src/main/java/com/netflix/spectator/jvm/Jmx.java

```diff
@@ -42,6 +44,15 @@ private Jmx() {
    * mbeans from the local jvm.
    */
   public static void registerStandardMXBeans(Registry registry) {
+    if (JavaFlightRecorder.isSupported()) {
+      Executor executor = Executors.newSingleThreadExecutor(r -> {
+        Thread t = new Thread(r, "spectator-jfr");
+        t.setDaemon(true);
+        return t;
+      });
+      JavaFlightRecorder.monitorDefaultEvents(registry, executor);
+      return;
+    }
     monitorClassLoadingMXBean(registry);
```

Comment:
Categories that are duplicated, like classloading and compilation stats, should only be done via JMX if not using JFR.

Reply:
Oops, yeah, meant to make that a condition. Fixed, good eye, thanks!